docs/development.rst

```diff
@@ -82,7 +82,6 @@ and isolating development in a dedicated environment as follows::
 
     $ conda create -q -n tskit-dev
     $ source activate tskit-dev
-    $ conda install -c conda-forge --yes --file=python/requirements/conda-minimal.txt
     $ conda install -c conda-forge doxygen
     $ conda install -c bioconda --yes pysam
```

Comment:
can't we install doxygen and pysam all at once with:
`conda -c conda-forge -c bioconda doxygen pysam`